site/content/en/docs/v0.1/quickstart.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Quickstart
+weight: 0
+description: Get etcd with etcd-operator up and running in less than 5 minutes!
+---
+
+Follow these instructions to install, run, and test etcd with etcd-operator in a Kubernetes cluster.
+
+Pre-requisites:
```

Comment:
While etcd-operator depends on cert-manager, it would be a good idea to add its installation to the prerequisites